src/services/stringCompletions.ts

```diff
@@ -8,10 +8,22 @@ namespace ts.Completions.StringCompletions {
         if (isInString(sourceFile, position, contextToken)) {
             if (!contextToken || !isStringLiteralLike(contextToken)) return undefined;
             const entries = getStringLiteralCompletionEntries(sourceFile, contextToken, position, checker, options, host);
-            return convertStringLiteralCompletions(entries, sourceFile, checker, log, preferences);
+            const completeInfo = convertStringLiteralCompletions(entries, sourceFile, checker, log, preferences);

```

Comment:
I would rather you do this inside the switch case of `StringLiteralCompletionKind.Properties` and  `StringLiteralCompletionKind.Types` to ensure we aren't alternating any returned entries.